maths/extended_euclidean_algorithm.py

```diff
@@ -3,69 +3,97 @@
 
 Finds 2 numbers a and b such that it satisfies
 the equation am + bn = gcd(m, n) (a.k.a Bezout's Identity)
+
+https://en.wikipedia.org/wiki/Extended_Euclidean_algorithm
 """
 
 # @Author: S. Sharma <silentcat>
 # @Date:   2019-02-25T12:08:53-06:00
 # @Email:  silentcat@protonmail.com
-# @Last modified by:   PatOnTheBack
-# @Last modified time: 2019-07-05
+# @Last modified by:   pikulet
+# @Last modified time: 2020-10-02
 
 import sys
 
 
-def extended_euclidean_algorithm(m, n):
+def sign(n: int) -> int:
```

Comment:
Is this function really necessary?

Reply:
I have made changes that does not require sign to do negative number correction